app/views/mailers/notifications_mailer/nudge_to_request_mentoring.html.haml

```diff
@@ -0,0 +1,10 @@
+%p Hi #{@user.handle},
+%p You've completed a few exercises (nice work!) but not yet requested mentoring on any of your solutions.
+%p
+  Receiving feedback from our mentors is an amazing way of discovering your knowledge gaps and levelling up your skills.
```

Comment:
Is it `levelling` in US English or `leveling`? I've done a quick search and found:

https://en.wikipedia.org/wiki/Levelling#:~:text=Levelling%20or%20leveling%20(American%20English,points%20relative%20to%20a%20datum.&text=It%20is%20also%20known%20as%20spirit%20levelling%20and%20differential%20levelling.
https://ell.stackexchange.com/questions/146181/in-us-english-which-is-the-appropriate-spelling-leveled-a-look-or-levelled-a-l

It _looks_ like it should be `leveling`, but 🤷

Reply:
I'm going to leave this one I think. One `l` just looks too wrong to me!!